### PR TITLE
feat: added namespace manifest

### LINKED ISSUE #24: Use a namespace manifest in istio demo
Use a namespace manifest in istio demo instead of a `kubectl` command.

### PR DESCRIPTION
**Pull request by swastik959**
added namespace manifest in the istio demo 

fixes #24

### CODE REVIEW

File: demo/istio/manifests/echo-service.yaml [hunk 1]
```diff
@@ -1,3 +1,12 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: my-istio-namespace
```
Comment (eddycharly) [inline demo/istio/manifests/echo-service.yaml:4]:
```suggestion
  name: demo
```

Comment (eddycharly) [reply demo/istio/manifests/echo-service.yaml:4]: you also need to add the namespace to the resources in the manifests

**Comment by eddycharly** [comment]
@swastik959 fix you pr title please

File: demo/istio/manifests/echo-service.yaml (hunk 1 above)
Comment (swastik959) [reply demo/istio/manifests/echo-service.yaml]: @eddycharly done

**Review by eddycharly — APPROVED** [review]
LGTM

**Pull request merged**